trunk/mailer.smtp.class.php

```diff
@@ -28,20 +26,10 @@ public function configure_phpmailer($phpmailer) {
         $phpmailer->SMTPSecure = 'tls';
         $phpmailer->Port = !empty($options['port']) ? intval($options['port']) : 587;
         $phpmailer->Host = 'smtp.sparkpostmail.com';
-        $phpmailer->AddReplyTo($options['from_email'], $options['from_name']); 
+//        $phpmailer->AddReplyTo($options['from_email'], $options['from_name']);
```

Comment:
we'll get rid of at all. we won't set Reply-To at all. Let me know if you think otherwise.